<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIW Database</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon/car.png</normaloff>../icon/car.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,0">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Select</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,3,0">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Project List </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_SelectProject">
           <property name="text">
            <string>select</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icon/select.png</normaloff>../icon/select.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_ClearQuery">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icon/clear.png</normaloff>../icon/clear.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Part</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_PartName">
           <property name="placeholderText">
            <string>零件号，如809609</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Thickness</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Thickness">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>厚度，如=1.5，&gt;0.8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Cladding</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Cladding">
           <property name="placeholderText">
            <string>镀层，如GI40/40</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Material</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Material">
           <property name="placeholderText">
            <string>材料名，如CR300</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Set">
           <property name="placeholderText">
            <string>如K1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Strength</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Strength">
           <property name="whatsThis">
            <string/>
           </property>
           <property name="accessibleDescription">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>强度等级，如=2，&gt;1,&lt;4</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Weight</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Weight">
           <property name="placeholderText">
            <string>重量（g），如&gt;1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Supplier</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Supplier">
           <property name="placeholderText">
            <string>供应商名称</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Class</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Class">
           <property name="text">
            <string>单件</string>
           </property>
           <property name="placeholderText">
            <string>单件、标准件或总成</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Pfeil</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Pfeil">
           <property name="placeholderText">
            <string>总成等级，如4,7Y</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,7">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="E_Name">
           <property name="placeholderText">
            <string>零件中文名或德文名</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Query</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="horizontalSpacing">
            <number>30</number>
           </property>
           <item row="1" column="0">
            <widget class="QPushButton" name="B_Query">
             <property name="text">
              <string>Query</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../icon/search.png</normaloff>../icon/search.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="C_AllItem">
             <property name="text">
              <string>All Items</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>User Define</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QPushButton" name="B_UserDefineImport">
                 <property name="text">
                  <string>Import</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../icon/import.png</normaloff>../icon/import.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="B_UserDefineSave">
                 <property name="maximumSize">
                  <size>
                   <width>1000</width>
                   <height>1000</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Save</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../icon/save.png</normaloff>../icon/save.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTextEdit" name="T_UserDefine"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QPushButton" name="B_UserDefine">
                 <property name="maximumSize">
                  <size>
                   <width>1000</width>
                   <height>1000</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Query</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../icon/search.png</normaloff>../icon/search.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="B_CancelUserDefine">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>../icon/clear.png</normaloff>../icon/clear.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>18</width>
                   <height>18</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="handleWidth">
        <number>2</number>
       </property>
       <widget class="QTabWidget" name="tabWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>100</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="Table">
         <attribute name="title">
          <string>Table</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
          <item>
           <widget class="QTableView" name="V_Output"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Number">
         <attribute name="title">
          <string>Number</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="V_Number_Layout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Weight">
         <attribute name="title">
          <string>Weight</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <layout class="QVBoxLayout" name="V_Weight_Layout"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Ratio">
         <attribute name="title">
          <string>Ratio</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="V_Ratio_Layout"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Message</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QTextBrowser" name="T_Message"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1120</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionManage"/>
   </widget>
   <addaction name="menuProject"/>
   <addaction name="menuUser"/>
  </widget>
  <action name="actionNewAccount">
   <property name="text">
    <string>NewAccount</string>
   </property>
   <property name="toolTip">
    <string>NewAccount</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="icon">
    <iconset>
     <normaloff>../icon/user.PNG</normaloff>../icon/user.PNG</iconset>
   </property>
   <property name="text">
    <string>User</string>
   </property>
   <property name="toolTip">
    <string>UserManagement</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset>
     <normaloff>../icon/import.png</normaloff>../icon/import.png</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>../icon/output.png</normaloff>../icon/output.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../icon/delete.png</normaloff>../icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionManage">
   <property name="text">
    <string>Manage</string>
   </property>
   <property name="toolTip">
    <string>UserManagement</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
